src/applications/disability-benefits/all-claims/tests/schema/transformedData.js

```diff
@@ -35,7 +35,6 @@ export const transformedMinimalData = {
         disabilityActionType: 'NONE',
       },
     ],
-    hasTrainingPay: false,
```

Comment:
Adding in the `view:claimType: { "view:claimingIncrease": true }` means this question never gets asked and consequently this data gets scrubbed out as its on an 'inactive' page.